components/helpSection/helpSection.scss

```diff
@@ -0,0 +1,44 @@
+@import 'include-media/dist/_include-media.scss';
+
+.HelpSection .HelpSection{
+  &-question{
+    line-height: 22px;
+    font-size: 14px;
+    font-weight: 500;
+    color: #1890ff;
+    margin-bottom: 5px;
+  }
+
+  &-issuesContributing{
+    font-size: 14px;
+    color: rgba(0,0,0,.65);
+
+    & p{
+      font-size: 14px;
+      margin-bottom: 0px;
+    }
+  }
+}
+
+.HelpSection{
+  font-weight: 400;
+  margin-bottom: 100px;
+  text-align: left;
+  margin-top: 30px;
+
+  .ant-collapse>.ant-collapse-item>.ant-collapse-header {
```

Comment:
No need to use `immediate children` selector (`>`). Ideally, we can specify everything enough with classes and elements. In the past, these kind of selectors were badly performant and might have got better now but better safe than sorry.